docs/docs/unexpected-input.mdx

```diff
@@ -146,96 +164,43 @@ slots:
       - job_function
 ```
 
-This means that Core will pay attention to the value of the slot when making a prediction
+This means that the dialogue model will pay attention to the value of the slot when making a prediction
 (read more about other [featurized slots](./domain.mdx#slots), whereas
-unfeaturized slots are only used for storing information. The stories for this should look as follows:
-
-```md
-## explain email
-* contact_sales
-    - sales_form
-    - active_loop{"name": "sales_form"}
-    - slot{"requested_slot": "business_email"}
-* explain
-    - utter_explain_why_email
-    - sales_form
-    - active_loop{"name": null}
-
-## explain budget
-* contact_sales
-    - sales_form
-    - active_loop{"name": "sales_form"}
-    - slot{"requested_slot": "budget"}
-* explain
-    - utter_explain_why_budget
-    - sales_form
-    - active_loop{"name": null}
-```
+unfeaturized slots are only used for storing information. 
 
-We'll need to add the intent and utterances we just added to our `domain.yml` file:
+You can then write stories for specific responses to interruptions based on the value of `requested_slot`, for example:
 
-```yaml
-intents:
-- greet: {triggers: action_greet_user}
-- bye
-- thank
-- faq
-- explain
-
-responses:
-  utter_explain_why_budget:
-  - text: We need to know your budget to recommend a subscription
-  utter_explain_why_email:
-  - text: We need your email so we can contact you
 ```
-
-Finally, we'll need to add some NLU data for the explain intent:
-
-```md
-## intent:explain
-- why
-- why is that
-- why do you need it
-- why do you need to know that?
-- could you explain why you need it?
+stories:
+- story: email interruption
+  steps:
+  - active_loop: sales_form
+  - slot_was_set: 
+    - requested_slot: email
+  - intent: ask_why_needed
+  - action: utter_why_email
+  - action: sales_form
+
+- story: company interruption
+  steps:
+  - active_loop: sales_form
+  - slot_was_set: 
+    - requested_slot: company
+  - intent: ask_why_needed
+  - action: utter_why_company
+  - action: sales_form
 ```
```

Comment:
@akelad this is the example I was wondering about. In a story, I'd have to start with `contact_sales` to make the whole interruption flow predictable right? I couldn't just keep it starting with active loop & slot?